### DISCUSSION TITLE
GraphML attributes

### DISCUSSION THREAD

**Discussion by leryss** [gephi/gephi#2394] [Q&A] [answered]
Is there any list of attributes that I can set in the .graphml format so that I can modify certain things in the graph ? For example I would like to set the font/font size/font color of the labels from the graphml file directly. I would also like if possible to specify a layout from the .graphml file

**Comment by mbastian** [comment, marked as answer]
Hi @leryss, 

We don't have that documented but it would be worth adding to here:https://gephi.org/users/supported-graph-formats/graphml-format/

Looking at the code, this is what we support for nodes:
- Node Label
  - label
  - nodelabel
- Position
  - x/xpos, y/ypos, z/zpos
- Size
  - size
- Color
  - r/g/b
  - color (hex)

Unfortunately, anything related to label styling isn't implemented at this point. Feel free to open a feature request if that's something that would help you.
